modules/ingress-certificates.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="ingress-certificates_{context}"]
+= Ingress certificates
+
+[discrete]
+== Purpose
+
+The Ingress Operator uses certificates for:
+
+* Securing access to metrics for Prometheus.
+* Securing access to routes.
+
+[discrete]
+== Location
+
+To secure access to Ingress Operator and Ingress Controller metrics, the Ingress
+Operator uses service serving certificates. The Operator requests a certificate
+from the `service-ca` controller for its own metrics, and the `service-ca`
+controller puts the certificate in a secret named `metrics-tls` in the
+`openshift-ingress-operator` namespace. Additionally, the Ingress Operator
+requests a certificate for each Ingress Controller, and the `service-ca`
+controller puts the certificate in a secret named `router-metrics-certs-<name>`,
+where `<name>` is the name of the Ingress Controller, in the
+`openshift-ingress` namespace.
+
+Each Ingress Controller has a default certificate that the Controller uses for
+secured routes that do not specify their own certificates. You can set a custom
+default certificate, or else the Operator generates a certificate for you. The
+Operator uses its own self-signed signing certificate to sign any default
```

Comment:
s/You can set a custom default certificate, or else the Operator generates a certificate for you/Unless you specify a custom certificate, the Operator uses a self-signed certificate by default.